server/routerlicious/packages/services-client/src/test/restwrapper.spec.ts

```diff
@@ -17,7 +17,7 @@ describe("RestWrapper", () => {
 
     before(() => {
         axiosMock = {
-            request: async (options?) => new Promise<AxiosResponse>(
+            request: async (options?) => new Promise<any>(
```

Comment:
The new version does not provide any type?

Reply:
The error that was being thrown here was along the lines of "AxiosResponse not assignable to type R; R can be an arbitrary type" because the `AxiosInstance.request` type declaration allows for alternate response types with the default set to `AxiosResponse` (`request<T = any, R = AxiosResponse>(...): Promise<R>`). I wonder if I could actually add T and R to this mock method...

Reply:
Updated to use <T, R> in the mock, but had to cast `response as any` in the resolve call